include/ck_tile/ops/gemm/pipeline/gemm_pipeline_agmem_bgmem_creg_v1_default_policy.hpp

```diff
@@ -11,6 +11,9 @@ namespace ck_tile {
 // Default policy class should not be templated, put template on member functions instead
 struct GemmPipelineAGmemBGmemCRegV1DefaultPolicy
 {
+
+    static constexpr index_t VectorLoadSize = 16;
```

Comment:
do we need to move this to the `TileGemmTraits<>`?

Reply:
Moved to the GemmTraits.